v2_1/ws/rest/docs/4_4_data_queries.md

```diff
@@ -1,91 +1,104 @@
+context | One of the following: `datastructure`, `dataflow`, `provisionagreement` | Data can be reported against a data structure, a dataflow or a provision agreement. This parameter allows selecting the desired context for data retrieval.
+agencyID | A string compliant with the SDMX *common:NCNameIDType* | The agency maintaining the artefact for which data have been reported. 
+resourceID | A string compliant with the SDMX *common:IDType* | The id of the artefact for which data have been reported.
+version | A string compliant with the *VersionType* defined in the SDMX Open API specification | The version of the artefact for which data have been reported.
+key | A string compliant with the *KeyType* defined in the SDMX Open API specification. | The combination of dimension values identifying the slice of the cube for which data should be returned. Wildcarding is supported via the `*` operator. For example, if the following key identifies the bilateral exchange rates for the daily US dollar exchange rate against the euro, D.USD.EUR.SP00.A, then the following key can be used to retrieve the data for all currencies against the euro: D.*.EUR.SP00.A.
 
 The parameters mentioned above are specified using the following syntax:
 
-    protocol://ws-entry-point/resource/flowRef/key/providerRef
-    
-Furthermore, some keywords may be used:
-
-Keywords | Scope | Description
---- | --- | ---
-all | key | Returns all data belonging to the specified dataflow and provided by the specified provider.
-all | providerRef | Returns all data matching the supplied key and belonging to the specified dataflow that has been provided by any data provider. As `all` is a reserved keyword in the SDMX RESTful API, it is recommended not to use it as an identifier for providers.
+    protocol://ws-entry-point/resource/context/agencyID/resourceID/version/key
 
 The following rules apply:
 
-- If no key is specified, all data (or metadata) belonging to the dataflow (or metadataflow) identified by the flowRef should be supplied. It is therefore equivalent to using the keyword `all`.
-- If no providerRef is specified, the matching data (or metadata) provided by any data provider should be returned. It is therefore equivalent to using the keyword `all`.
+- All parameters support multiple values, using comma (`,`) as separator.
+- All parameters offer the option to retrieve all existing values, using the `*` operator.
+- Two additional operators are supported for the version parameter: the `+`, to indicate the latest stable version of an artefact, and `~`, to indicate the latest version of an artefact regardless of its status (draft vs. stable). 
+- For all path parameters, the default value is `*`.
```

Comment:
Except 'resource', 'context' and 'resourceID'.

Reply:
See previous comment :)